test/e2e/azure_test.go

```diff
@@ -100,7 +100,7 @@ var _ = Describe("Workload cluster creation", func() {
 	})
 
 	Context("Create multiple controlplane cluster with machine deployments", func() {
```

Comment:
should we also update the context to be more accurate?

Reply:
updated, thanks!